README.md

```diff
@@ -37,16 +37,166 @@ NOTE: we suggest using virtualenv to sandbox your setup.
 - [VALIDATOR_GUIDE.md](https://github.com/ethereum/casper/blob/master/VALIDATOR_GUIDE.md):
 information about implementing a Casper FFG validator.
 
-## Contribute
+## Tests
 
-Feel free to ask questions in our [Gitter room](https://gitter.im/ethereum/casper) or open an [issue](https://github.com/ethereum/casper/issues) for feature requests or bug reports. Feel free to make a PR!
+```bash
+pytest tests
+```
 
-## License
+## Version of Vyper to Compile Casper
+We recommend using the vyper version installed as a dependency from requirements.txt. 
+NOTE: The latest version of Vyper throws an error (line 16) when compiling the Casper contract due to a syntax change of "units".
+NOTE: this not needed if steps above followed
+```bash
+git clone https://github.com/ethereum/vyper.git@248c723288e84899908048efff4c3e0b12f0b3dc
+```
+The guide on how to install Vyper can be found [here](https://github.com/ethereum/vyper#installation).
 
-[UNLICENSE](LICENSE)
 
-## Tests
+## How to Build and Deploy Casper Contract
+
+In this example we will be using [eth-tester](https://github.com/ethereum/eth-tester).
+
 
+### Step 1: Imports
```

Comment:
I want this to be way less system specific. Think, high level notes and a little bit of pseudocode. So we don't need to deal with imports